src/main/java/uk/gov/hmcts/reform/bulkscanprocessor/config/QueueClientConfig.java

```diff
@@ -0,0 +1,50 @@
+package uk.gov.hmcts.reform.bulkscanprocessor.config;
+
+import com.microsoft.azure.servicebus.IQueueClient;
+import com.microsoft.azure.servicebus.QueueClient;
+import com.microsoft.azure.servicebus.ReceiveMode;
+import com.microsoft.azure.servicebus.primitives.ConnectionStringBuilder;
+import com.microsoft.azure.servicebus.primitives.ServiceBusException;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.context.annotation.Profile;
+
+@Configuration
+@Profile("!nosb") // only active when Service Bus interactions aren't disabled
+public class QueueClientConfig {
+
+    @Bean("envelopes")
+    public IQueueClient envelopesQueueClient(
+        @Value("${queues.envelopes.connection-string}") String connectionString,
+        @Value("${queues.envelopes.queue-name}") String queueName
+    ) throws InterruptedException, ServiceBusException {
+        return createQueueClient(connectionString, queueName);
+    }
+
+    @Bean("notifications")
+    public IQueueClient notificationsQueueClient(
+        @Value("${queues.notifications.connection-string}") String connectionString,
+        @Value("${queues.notifications.queue-name}") String queueName
+    ) throws InterruptedException, ServiceBusException {
+        return createQueueClient(connectionString, queueName);
+    }
+
+    @Bean("read-notifications")
```

Comment:
So we have `notifications` and `read-notifications`.
Is the first on for both reading and writing. Or just writing?
Worth clarifying imho.

Reply:
I was following the existing convention - notifications vs read-notifications. Probably best to leave it as it is and change everything in a separate PR to let this one address one thing.